traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/dns/RRSIGCacheKey.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.comcast.cdn.traffic_control.traffic_router.core.dns;
+
+import java.util.Arrays;
+
+public class RRSIGCacheKey {
+    final private byte[] privateKeyBytes;
+    final private int algorithm;
+
+    public RRSIGCacheKey(final byte[] privateKeyBytes, final int algorithm) {
+        this.privateKeyBytes = privateKeyBytes;
+        this.algorithm = algorithm;
+    }
+
+    public byte[] getPrivateKeyBytes() {
+        return privateKeyBytes;
+    }
+
+    public int getAlgorithm() {
+        return algorithm;
+    }
```

Comment:
`getAlgorithm()` looks unused, too. A breakpoint I set inside it was never tripped.

Reply:
You're right -- I'll remove